WordPress/Classes/ViewRelated/Domains/Domain registration/RegisterDomainSuggestions/RegisterDomainSuggestionsViewController.swift

```diff
@@ -25,12 +25,6 @@ class RegisterDomainSuggestionsViewController: UIViewController {
     private var includeSupportButton: Bool = true
     private var navBarTitle: String = TextContent.title
 
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        configure()
-        hideButton()
-    }
```

Comment:
I didn't really remove this `viewDidLoad`. I just moved to line 93 and added `setupTransferFooterView()`.